### PR TITLE
Adding Example 2 to illustrate NotificationGroup parameter 

### PR DESCRIPTION
**Pull request by DEberhardt**
Set-CsTeamsEmergencyCallingPolicy has a good example on how to use the NotificationGroup parameter.
Copying this example here as Example 2

Addresses #9653

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ PS C:>   New-CsTeamsEmergencyCallingPolicy -Identity testECP -Description "Test
 
  This example creates a Teams Emergency Calling policy that has a identity of testECP, where a notification group and number is being defined, the external location lookup mode is enabled and also the type of notification.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md:38]:
```suggestion
 This example creates a Teams Emergency Calling policy that has an identity of testECP, where a notification group and number are being defined, the external location lookup mode is enabled and also the type of notification.
```

File: skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md [hunk 2]
```diff
@@ -37,6 +37,13 @@ PS C:>   New-CsTeamsEmergencyCallingPolicy -Identity testECP -Description "Test
...
 
+### Example 2
+```powershell
+PS C:>  New-CsTeamsEmergencyCallingPolicy -Identity "testECP2" -NotificationGroup "123@gh.com;567@test.com"
+```
+
+ This example creates a Teams Emergency Calling policy that has a identity of testECP2, with default settings except for the Notification Group. This parameter expects a single string with all users and groups separated with ";".
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md:45]:
```suggestion
 This example creates a Teams Emergency Calling policy that has an identity of testECP2, with default settings except for the Notification Group. This parameter expects a single string with all users and groups separated with ";".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DEberhardt, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**